frontend/src/pages/settings/server.vue

```diff
@@ -0,0 +1,255 @@
+
+  try {
+    const configApi = remote.sdk.newUserApi(getConfigurationApi);
+    const configRes = await configApi.getConfiguration();
+    const config = configRes.data;
+
+    config.ServerName = serverSettings.value.state.value.name;
+    config.UICulture = serverSettings.value.state.value.language;
+    config.QuickConnectAvailable = serverSettings.value.state.value.quickConnect;
+    config.CachePath = serverSettings.value.state.value.cachePath;
+    config.MetadataPath = serverSettings.value.state.value.metadataPath;
+    config.LibraryScanFanoutConcurrency = serverSettings.value.state.value.parallelLibraryScan;
+    config.ParallelImageEncodingLimit = serverSettings.value.state.value.parallelImageEncoding;
+
+    const brandingApi = remote.sdk.newUserApi(getBrandingApi);
+    const brandingRes = await brandingApi.getBrandingOptions();
+    const branding = brandingRes.data;
+
+    branding.LoginDisclaimer = serverSettings.value.state.value.loginDisclaimer;
+    branding.CustomCss = serverSettings.value.state.value.customCSS;
+    branding.SplashscreenEnabled = serverSettings.value.state.value.enableSplash;
+
+    await configApi.updateConfiguration({
+      serverConfiguration: config
+    });
+    await configApi.updateNamedConfiguration({
+      key: 'branding',
+      body: JSON.stringify(branding)
+    });
+
+    console.log('Settings saved successfully');
+  } catch (error) {
+    console.error('Error saving settings:', error);
+  } finally {
+    saving.value = false;
+  }
+}
+
+onMounted(async () => {
+  loading.value = true;
+
+  try {
+    const localizationApi = remote.sdk.newUserApi(getLocalizationApi);
+
+    culturesList.value = (
+      await localizationApi.getLocalizationOptions()
+    ).data;
+
+    const configApi = remote.sdk.newUserApi(getConfigurationApi);
+    const configRes = await configApi.getConfiguration();
+    const config = configRes.data;
+
+    serverSettings.value.state.value.name = config.ServerName;
+    serverSettings.value.state.value.language = config.UICulture;
+    serverSettings.value.state.value.quickConnect = config.QuickConnectAvailable;
+    serverSettings.value.state.value.cachePath = config.CachePath;
+    serverSettings.value.state.value.metadataPath = config.MetadataPath;
+    serverSettings.value.state.value.parallelLibraryScan = config.LibraryScanFanoutConcurrency;
+    serverSettings.value.state.value.parallelImageEncoding = config.ParallelImageEncodingLimit;
+
+    const brandingApi = remote.sdk.newUserApi(getBrandingApi);
+    const brandingRes = await brandingApi.getBrandingOptions();
+    const branding = brandingRes.data;
+
+    serverSettings.value.state.value.loginDisclaimer = branding.LoginDisclaimer;
+    serverSettings.value.state.value.customCSS = branding.CustomCss;
+    serverSettings.value.state.value.enableSplash = branding.SplashscreenEnabled;
+  } catch (error) {
+    console.error('Error loading settings:', error);
+  } finally {
+    loading.value = false;
+  }
+});
```

Comment:
Honestly not sure how I feel about this code, was considering changing it to a hook or abstracting it away somewhere else, thought id get opinions first though